.circleci/config.yml

```diff
@@ -0,0 +1,202 @@
+version: 2.1
+
+orbs:
+  slack: circleci/slack@3.4.2
+
+parameters:
+  api_triggered:
+    type: boolean
+    description: Whether the pipeline was triggered through the CircleCi API (https://circleci.com/docs/api/v2/?shell#trigger-a-new-pipeline).
```

Comment:
Minor: We should link here to `docs/manual-building.md`.

Reply:
That doc is not yet available, right?

Reply:
I'd also do documentation in another task as soon as you docu task is merged.

Reply:
You can link here to the nonexisting doc. Else we may forget it.

Reply:
I won't forget it, I'll do it in a separate docu task once your PR is merged. I put it on my list.